### PR TITLE
New App: TodoistNext

### PR DESCRIPTION
**Pull request by akeslo**
## TodoistNext
Displays the next due/overdue task from Todoist.  
https://todoist.com/ Integration.

Base of code written by [alisdair](https://discuss.tidbyt.com/u/alisdair).  See - https://discuss.tidbyt.com/t/todoist-integration/502/6

### CODE REVIEW

**Comment by akeslo** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by akeslo** [comment]
recheck

File: apps/todoistnext/todoist_next.star [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+TODOIST_API_BASE_URL = "https://api.todoist.com/rest/v1"
+
+TODOIST_API_TASKS_URL = TODOIST_API_BASE_URL + "/tasks?filter=(overdue|today)"
+
+MODEL_KEY_TEXT = "text"
+MODEL_KEY_DUE = "due"
+MODEL_KEY_ZEN = False
+
+CACHE_KEY_MODEL = "todoist_model"
```
Comment (matslina) [inline apps/todoistnext/todoist_next.star:24]: Was the intention to add caching somewhere? Not sure if it's needed, but probably would be appreciated by the API provider.

File: apps/todoistnext/todoistnext.go [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+//go:embed todoist_next.star
+var source []byte
+
+// New creates a new instance of the Todoist Next applet.
+func New() manifest.Manifest {
+	return manifest.Manifest{
+		ID:          "todoist-next",
+		Name:        "Todoist Next",
+		Author:      "Alisdair/Akeslo",
```
Comment (matslina) [inline apps/todoistnext/todoistnext.go:18]: Did you hear back from the original author? Not sure if we really can merge it without their OK.

**Review by matslina — CHANGES_REQUESTED** [review]

File: apps/todoistnext/todoistnext.go (hunk 2 above)
Comment (akeslo) [reply apps/todoistnext/todoistnext.go:18]: hmm ok, I did not hear back. I'll ping again and if no response I can just rewrite and include caching.

File: apps/todoistnext/todoist_next.star (hunk 1 above)
Comment (akeslo) [reply apps/todoistnext/todoist_next.star:24]: see below

File: apps/todoistnext/todoistnext.go (hunk 2 above)
Comment (AlisdairGurling) [reply apps/todoistnext/todoistnext.go:18]: Hi, I am the original author. Please go ahead and well done!

Comment (matslina) [reply apps/todoistnext/todoistnext.go:18]: Sweet!

**Review by matslina — APPROVED** [review]

**Pull request merged**